packages/ckeditor5-essentials/src/essentials.ts

```diff
@@ -36,7 +37,7 @@ export default class Essentials extends Plugin {
 	 * @inheritDoc
 	 */
 	public static get requires() {
-		return [ Clipboard, Enter, SelectAll, ShiftEnter, Typing, Undo ] as const;
+		return [ Clipboard, Enter, SelectAll, ShiftEnter, Typing, Undo, AccessibilityHelp ] as const;
```

Comment:
I don't like the way it gets injected. `Essentials` is just a plugin not every integration will use it. OTOH I couldn't just use the same [approach](https://github.com/ckeditor/ckeditor5/blob/e3b93bfdd65b8756519812576884434362add1d1/packages/ckeditor5-ui/src/editorui/editorui.ts#L138-L139) as with `PoweredBy` or `Tooltip` because these are not plugins (`AccessibilityHelp` is a plugin and it requires `Dialog`).

I'm wondering if it would be OK to force-inject `AccessibilityHelp` into `editor.config.plugins` at some point? [`BalloonEditor` does that](https://github.com/ckeditor/ckeditor5/blob/e3b93bfdd65b8756519812576884434362add1d1/packages/ckeditor5-editor-balloon/src/ballooneditor.ts#L91-L92) (but it's an editor class). Optimally that would take place in `EditorUI` but there are a few gotchas:
* not sure if the timing is right (would need to check when `EditorUI` gets created)
* not sure if all integrators want the `AccessibilityHelp` feature (?), if only because it requires `Dialog`.

Reply:
If we split this functionality between `AccessibilityDialog` and some registry for the accessibility help info, maybe it will make more sense to add `AccessibilityDialog` to the `Essentials` plugin. Buuuut, AFAICS, neither of essentials plugin requires our UI (or UI at all). So maybe we shouldn't change this.

I am not happy with force-adding the plugin to the `Editor` class or somewhere through the `EditorUI` class. It doesn't feel good.

Side note: requiring the `Dialog` plugin just in itself is not a big problem but yes, it makes other things into plugins. Maybe it is a mistake to have dialog functionality as a plugin and it should have been available through `editor.ui`?

Back to the question. Do we actually need to force the a11y dialog upon people? Maybe let's just have it as any other plugin?

If we want to automate and we are worried about different kinds of editors, we could add a11y dialog plugin to the specific editor classes that we have. At least we know that all of them have UIs.

Reply:
F2F decision:
* Let's go with `Essentials` because, for instance, unlike the balloon toolbar in `BalloonEditor`, this is not a critical functionality for every editor, and the coupling would be too tight if we force-injected it in editor constructors. Besides, this option is still on the table.
* Let's keep in mind that we need to have `Essentials` or `AccessibilityHelp` in snippets (guides). This will be a follow-up.
* Let's check the compatibility with context plugins (a follow-up).